scripts/test_runner.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+ENV_PRODUCT="${ENV_PRODUCT}"
+ENV_TFVARS="${ENV_TFVARS}"
+TESTDIR="${TESTDIR}"
+TESTTAG="${TESTTAG}"
+CMD="${CMD}"
+EXPECTEDVALUE="${EXPECTEDVALUE}"
+VALUEUPGRADED="${VALUEUPGRADED}"
+CHANNEL="${CHANNEL}"
+INSTALLVERSIONORCOMMIT="${INSTALLVERSIONORCOMMIT}"
+SUCUPGRADEVERSION="${SUCUPGRADEVERSION}"
+TESTCASE="${TESTCASE}"
+WORKLOADNAME="${WORKLOADNAME}"
+DESCRIPTION="${DESCRIPTION}"
+DEPLOYWORKLOAD="${DEPLOYWORKLOAD}"
+SONOBUOYVERSION="${SONOBUOYVERSION}"
```

Comment:
Do we need to redefine all of these variables?

Reply:
Not mandatory. I was just checking if has value in it and "documenting" in the beginning.  Take it out?